tests/test_hoomd.py

```diff
@@ -550,3 +551,17 @@ def test_log(tmp_path, open_mode):
         # specified entries are different in frame 1
         numpy.testing.assert_array_equal(s.log['particles/pair_lj_energy'], snap1.log['particles/pair_lj_energy']);
         numpy.testing.assert_array_equal(s.log['value/pressure'], snap1.log['value/pressure']);
+
+def test_pickle(tmp_path, open_mode):
+    with gsd.hoomd.open(name=tmp_path / "test_pickling.gsd",
+                        mode=open_mode.write) as traj:
+        traj.extend((create_frame(i) for i in range(20)))
+        with pytest.raises(pickle.PickleError):
+            pkl = pickle.dumps(traj)
+    with gsd.hoomd.open(name=tmp_path / "test_pickling.gsd",
+                        mode=open_mode.read) as traj:
+        pkl = pickle.dumps(traj)
+        hf = pickle.loads(pkl)
```

Comment:
Can you change this to `with hf` so that you don't have to manually close?

Reply:
Can and did